specification/quantum/data-plane/Microsoft.Quantum/preview/2021-05-06-preview/quantum.json

```diff
@@ -0,0 +1,757 @@
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/JobDetailsList"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get list of jobs in a Quantum Workspace": {
+            "$ref": "./examples/jobList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Quantum/workspaces/{workspaceName}/jobs/{jobId}": {
+      "get": {
+        "operationId": "Jobs_Get",
+        "description": "Get job by id",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/jobIdParameter"
+          }
+        ],
+        "x-ms-examples": {
+          "Get a job's metadata": {
+            "$ref": "./examples/jobGet.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/JobDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Jobs_Create",
+        "description": "Create a job.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/jobIdParameter"
+          },
+          {
+            "name": "job",
+            "description": "The complete metadata of the job to submit.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/JobDetails"
+            }
+          }
+        ],
+        "x-ms-examples": {
+          "Submit a new job for execution to a Quantum Workspace": {
+            "$ref": "./examples/jobPut.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/JobDetails"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/JobDetails"
+            }
```

Comment:
If this is a long-running operation apply the annotation https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-long-running-operation

Reply:
skipping this for now because I don't know the service impact of that annotation (and it's unrelated to the primary change in this PR).